tests/ieee802154_hal/Makefile

```diff
@@ -21,14 +21,16 @@ DISABLE_MODULE += auto_init_at86rf2xx auto_init_nrf802154
 
 USEMODULE += od
 USEMODULE += luid
+USEMODULE += l2util
```

Comment:
For some reason, this is not included for `remote-revb` in the CI...

Reply:
Lets run again? I can't reproduce locally...

Reply:
If the test is flaky, don't we maybe want to wait with merging this, _after_ feature freeze? Or is this PR critical for the release?

Reply:
Nope its a cleanup, it can wait